src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <Version>1.0.1</Version>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <PackagePlatforms>x64;x86;arm</PackagePlatforms>
+    <OutputPath>$(PackagesOutputPath)</OutputPath>
+  </PropertyGroup>
+  
+  <!-- TODO: Add Reference to Apiset Nuget package -->
```

Comment:
For now you can have 
```
<Dependency Include="Microsoft.NETCore.Windows.ApiSets">
  <Version>1.0.1-rc3-23803</Version>
</Dependency>
```
For this one maybe we should just produce a stable version early and depend on that.
Alternatively we could build the package here, but then we'd still need to get the actual api set dlls somehow.